src/RemixDevTools/RemixDevTools.tsx

```diff
@@ -23,7 +23,9 @@ interface Props {
 }
 
 const RemixDevTools = ({ defaultOpen, position }: Props) => {
-  const [isOpen, setIsOpen] = useState(defaultOpen);
+  const [isOpen, setIsOpen] = useState<boolean>(
+    defaultOpen || JSON.parse(localStorage.getItem("rdt-open") || "false")
```

Comment:
I see the point but this is not the way to do it, if you look at the context that wraps the whole app it saves all the user settings to localStorage and all temporary stuff to session storage, we should use the context here and let it handle the storage without us having to worry about it. Let me think about this a bit, if defaultOpen is set it should respect that if the rule is there on startup but the following times it should listen to this